spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/asset/Asset.java

```diff
@@ -48,91 +51,73 @@ public String getVersion() {
         return version;
     }
 
-    public String getFileName() {
-        return fileName;
-    }
 
     public Integer getByteSize() {
         return byteSize;
     }
 
-    // TODO in theory each asset may have multiple representations/media types, depending on the transfer capabilities of the EDC
-    public String getFileExtension() {
```

Comment:
If we remove the file extension from the Asset, there is no way to describe the IDS Representation. In this cast it's just a class without any set properties.

Reply:
Ok once again. If IDS requires a file extension or a byte size, that's a bug in the spec and it will be changed. The reason for this is because streams don't **have** file names or byte sizes and we can't send erroneous information down.

Reply:
So, if we don't add these metadata to the labels and still delete them. You suggest to not describe the IDS representation and artifact (except byte size) until they change their spec?